docs/source/algorithms/FitPeaks-v1.rst

```diff
@@ -137,23 +138,18 @@ There are two input parameters that are associated with fitting window.
 * FitPeakWindowWorkspace
 
 
-If FitWindows is defined, then a peak's range to fit (i.e., x-min and
+If either of these is defined, then a peak's range to fit (i.e., x-min and
 x-max) is confined by this window.
 
-If FitWindows is defined, starting peak centres are NOT user's input,
-but found by highest value within peak window. (Is this correct???)
-
-
-Further down the road, here are the fitting setup that can be affected.
-
-* Peak positions are uniform among all spectra
-
-  - Peak window information will be retrieved from **m_peakWindowVector**
-
-* Peak positions are different among spectra.
-
-  - Peak windown information will be retrieved from **m_peakWindowWorkspace**
+The fit windows must correspond to a peak center.  For each peak center, the window list must
+have a lower and upper bound, meaning each window list must be twice as long as the corresponding center list.
 
+If the windows are specified using FitWindowBoundaryList, then the same set of windows will
+be used on all spectra to be fit in the workspace.
+In this case window information will be retried from **m_peakWindowVector**.
```

Comment:
Use the parameter names given to the user in the docs rather than the member variable names. Most people don't want to dig into the source code.

Reply:
Good call